racket-doc.el

```diff
@@ -60,11 +60,15 @@ Centralizes how to issue doc command and handle response correctly."
 
 (defun racket--search-doc-locally (str)
   (racket--doc-assert-local-back-end)
-  (call-process racket-program
-                nil ;INFILE: none
-                0   ;DESTINATION: discard/don't wait
-                nil ;DISPLAY: none
-                "-l" "raco" "docs" str))
+  (let ((command (if (stringp racket-program)
+                     (list racket-program)
+                   racket-program)))
+    (apply call-process `(,(car command)
+                          nil ;INFILE: none
+                          0   ;DESTINATION: discard/don't wait
+                          nil ;DISPLAY: none
+                          ,@(cdr command)
+                          "-l" "raco" "docs" str))))
```

Comment:
We need to unquasiquote `str` here, i.e. add a `,` before it. That's what the byte compiler is complaining about on CI: ` racket-doc.el:61:1:Warning: Unused lexical argument ‘str’`.